### PR TITLE
Feat: add error handling

### LINKED ISSUE #31: Error Handling
Create Error Handling - for status codes

### PR DESCRIPTION
**Pull request by GOECHA**
Error Handling
- Created two functional components for error handling. 
- Created a dynamic error handling functional component "Status"
- Created a new functional component "NotFound" for 404 error handling
- NotFound is imported in App and rendered below containers
- Please see comments directly in code where changes have been made. 

- [x] New feature (adds functionality)
- [x] # How Has This Been Tested?
​ - tested in the DOM

Please describe the tests that you ran to verify your changes.
​
- [x] I tested my changes in the browser
- [] I tested with Mocha/Chai
​
# Checklist:
​
- [] My code follows the style guidelines of this project
  - Still in Development. There are potential design directions we can go in for redirect pages
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new bugs
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -6,8 +6,9 @@ import MovieContainer from '../MovieContainer/MovieContainer';
 import './App.css';
 import { getAllData } from '../../api-calls';
 import SingleMovie from '../Modal/Modal';
-import { Route } from 'react-router-dom'
+import { Route } from 'react-router-dom';
 // import Carousel from '../Carousel/Carousel';
+import NotFound from '../NotFound/NotFound';
```
Comment (GOECHA) [inline src/components/App/App.js:13]: Import from functional Component

File: src/components/App/App.js [hunk 2]
```diff
@@ -22,13 +23,16 @@ class App extends Component {
...
+
+
       
   render = () => {
     return (
       <main className = 'app'>
         <Header />
         <Route exact path='/' render={ () => <MovieContainer movies={this.state.movies}/> } />
         <Route exact path='/modal/:id' render={ ({match}) => <SingleMovie id={match.params.id} /> } />
+        <Route component={NotFound} />
```
Comment (GOECHA) [inline src/components/App/App.js:36]: Rendered functional component for 404 notfound error handling

File: src/components/NotFound/NotFound.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import Status from '../Status/Status';
+
+
+
+
+
+const NotFound = () => {
```
Comment (GOECHA) [inline src/components/NotFound/NotFound.js:8]: Functional component for 404 error handling

File: src/components/Status/Status.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Route } from 'react-router-dom';
+
+
```
Comment (GOECHA) [inline src/components/Status/Status.js:4]: Dynamic Functional Component for error handling.

Comment (tfettig22) [reply src/components/Status/Status.js:4]: Nice! this will be useful for updating our UI when errors occur

**Review by tfettig22 — APPROVED** [review]
Looks good, thanks for getting this up and running!

**Pull request merged**